packages/itmat-job-executor/src/utils/jobUtils.ts

```diff
@@ -1,32 +1,15 @@
-import { Logger } from 'itmat-utils';
-import mongodb from 'mongodb';
-
-export class JobUtils {
-    constructor(private readonly jobCollection: mongodb.Collection) {}
-
-    public async setJobError(jobId: string, msg: string) {
-        try {
-            await this.jobCollection.updateOne({ id: jobId }, { $set: {
-                error: msg,
-                status: 'TERMINATED WITH ERROR'
-            }});
-        } catch (e) {
-            Logger.error(`Cannot set job ${jobId} with error "${msg}"`);
-        }
-    }
-}
+import { IFieldDescriptionObject } from 'itmat-commons/dist/models/data';
```

Comment:
let's not use deep import when writing new codes. we ll make a pr to deal with the old ones as well

Reply:
At this point little to be done here until that change is brought:
![image](https://user-images.githubusercontent.com/2495155/83977499-c9401f80-a8f8-11ea-9e5c-eae234fc9b08.png)

Reply:
```javascript
import { Models } from 'itmat-commons';
type IFieldDescriptionObject = Models.Data.IFieldDescriptionObject;
```